puppetserver/docker-entrypoint.d/86-setup-enc.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+if [ -n "$PUPPET_NODE_TERMINUS" ]; then
+  puppet config set node_terminus $PUPPET_NODE_TERMINUS --section master
+fi
+
+if [ -n "$PUPPET_EXTERNAL_NODES" ]; then
+  puppet config set external_nodes $PUPPET_EXTERNAL_NODES --section master
```

Comment:
`master` is deprecated please user server

Reply:
Done. I used that because a lot of the other scripts use it as well